conans/client/build/autotools_environment.py

```diff
@@ -164,12 +164,12 @@ def configure(self, configure_dir=None, args=None, build=None, host=None, target
 
         if pkg_config_paths:
             pkg_env = {"PKG_CONFIG_PATH":
-                       [os.pathsep.join(get_abs_path(f, self._conanfile.install_folder)
+                       [os.pathsep.join(get_abs_path(f, self._conanfile.generators_folder)
```

Comment:
I cannot find the place where this is defined, and that by default should be the same folder it was before. I'd like to be sure of this.